test/models/wager_test.exs

```diff
@@ -3,7 +3,7 @@ defmodule MovieWagerApi.WagerTest do
 
   alias MovieWagerApi.Wager
 
-  @valid_attrs %{amount: 42, place: 42}
+  @valid_attrs %{amount: 42, place: 42, user_id: 50, movie_round_id: 55}
```

Comment:
🤔  Then I must have missed something, as this wasn't working.  Oh well, will get it right the next time around